code/modules/mob/living/living.dm

```diff
@@ -242,13 +246,15 @@
 /mob/living/start_pulling(atom/movable/AM, state, force = pull_force, supress_message = FALSE)
 	if(!AM || !src)
 		return FALSE
+
+	SEND_SIGNAL(AM, COMSIG_MOVABLE_ATTEMPT_PULL, src)
+	AM.add_fingerprint(src)
```

Comment:
Maybe try going with the `PRE` / `POST` naming convention?
That said, the order of the checks here should change, this is adding fingerprints even if the person is unable to pull, such as while paralyzed.
Should be something like this:
```
	if(throwing || !(mobility_flags & MOBILITY_PULL))
		return FALSE

	if(!(AM.can_be_pulled(src, state, force)))
		return FALSE

	SEND_SIGNAL(AM, COMSIG_MOVABLE_ATTEMPT_PULL, src)

	AM.add_fingerprint(src)
```

Reply:
That's kind of the thing though, I want this to run even if you can't pull.  I might separate the can_be_pulled check into two procs because what you're saying makes sense; you shouldnt be unconscious etc. But can be pulled fails if its in your hand, And the interaction this is used for is specifically an in-hand one.

Reply:
thats kind of why I went with "attempt pull" rather than "pre pull" if that makes sense?